src/Deku/Core.purs

```diff
@@ -231,7 +203,7 @@ type SendToPos =
 derive instance Newtype (DOMInterpret payload) _
 
 newtype DOMInterpret payload = DOMInterpret
-  { ids :: Zora String
+  { ids :: ST Global String
```

Comment:
This is made `ST Global` to accommodate for SSR, right?

Reply:
Yup!